examples/market/marketbot.rb

```diff
@@ -1,24 +1,29 @@
+require 'iex-ruby-client'
 require 'slack-ruby-bot'
-require 'yahoo-finance'
 
 SlackRubyBot::Client.logger.level = Logger::WARN
 
 class MarketBot < SlackRubyBot::Bot
   scan(/([A-Z]{2,5}+)/) do |client, data, stocks|
-    YahooFinance::Client.new.quotes(stocks, %i[name symbol last_trade_price change change_in_percent]).each do |quote|
-      next if quote.symbol == 'N/A'
-      client.web_client.chat_postMessage(
-        channel: data.channel,
-        as_user: true,
-        attachments: [
-          {
-            fallback: "#{quote.name} (#{quote.symbol}): $#{quote.last_trade_price}",
-            title: "#{quote.name} (#{quote.symbol})",
-            text: "$#{quote.last_trade_price} (#{quote.change_in_percent})",
-            color: quote.change.to_f > 0 ? '#00FF00' : '#FF0000'
-          }
-        ]
-      )
+    stocks.each do |stock|
+      begin
+        quote = IEX::Resources::Quote.get(stock.first)
```

Comment:
Note https://github.com/dblock/iex-ruby-client/pull/42. In June this stops working. So you got another PR on the tail of this to make :)